### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by zach-garno**
Adding the read me with procedures.

### CODE REVIEW

**Review by asoriano12 — APPROVED** [review]

**Review by caseyjkey — APPROVED** [review]
markdown is cool. We switched to a 20700 battery + battery sled + battery boost/charger module which I had at home. This allows us to have a more modular system which can be upgraded later to fit design changes.

**Comment by caseyjkey** [comment]
i do not have write access and cannot merge

**Comment by zach-garno** [comment]
I checked the permissions, and it shows you as still waiting for response. Look for the email invite and 'accept?' the invite?





________________________________

**Pull request merged**
